src/components/Menu/index.tsx

```diff
@@ -1,4 +1,7 @@
+"use client"
```

Comment:
Is it possible to move this to inside the `Dropdown` component?

Reply:
done